cpp/src/strings/repeat_strings.cu

```diff
@@ -189,146 +189,63 @@ struct compute_size_and_repeat_separately_fn {
   // If d_chars != nullptr: only repeat strings.
   char* d_chars{nullptr};
 
-  __device__ int64_t operator()(size_type const idx) const noexcept
+  __device__ void operator()(size_type const idx) const noexcept
```

Comment:
is this a `const` function still?

Reply:
Yes, the member variables are not changed by this function.

Reply:
should writes to `d_offsets[idx]` and `output_ptr` be not considered for this?

Reply:
https://stackoverflow.com/questions/7390437/c-const-member-functions-are-modifying-member-variables

got it.

Reply:
That's correct. The function does not change the actual `d_offsets` or `d_chars` variables. Changing data/memory they point to does not count.